### DISCUSSION TITLE
Does the Image component not support loading images from the internet?

### DISCUSSION THREAD

**Discussion by hellorust2056** [slint-ui/slint#7130] [Q&A] [answered]
Does the Image component not support loading images from the internet?

export component App inherits Window {
    width: 200px;
    height: 200px;
    VerticalLayout {
        Image {
            source: @image-url("https://slint.dev/logo/slint-logo-full-light.svg");
            // image-fit default is `contain` when in layout, preserving aspect ratio
        }
        Image {
            source: @image-url("https://slint.dev/logo/slint-logo-full-light.svg");
            colorize: red;
        }
    }
}

**Comment by tronical** [comment, marked as answer]
That is indeed not supported, see also #3279 .
